libsolidity/inlineasm/AsmPrinter.cpp

```diff
@@ -37,11 +37,24 @@ using namespace dev::solidity;
 using namespace dev::solidity::assembly;
 
 //@TODO source locations
+namespace {
+
+string appendLocation(SourceLocation const& _location)
+{
+	if (_location.isEmpty())
+		return "";
+	return
+		"/// @location " + *_location.sourceName +
```

Comment:
Please quote the source name, we don't know what we have to deal with in the future.
Also wouldn't `file:start-end` make more sense?